images/kindnetd/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # first stage build kindnetd binary
 # NOTE: the actual go version will be overridden
-FROM --platform=$BUILDPLATFORM golang:latest
+FROM --platform=$BUILDPLATFORM docker.io/golang:latest
```

Comment:
Shouldn't this be:
```suggestion
FROM --platform=$BUILDPLATFORM docker.io/library/golang:latest
```

Reply:
Technically perhaps yes. But at https://hub.docker.com/r/library/golang (which redirects to https://hub.docker.com/_/golang) the command shown is `docker pull golang`, not `docker pull library/golang`. I will change it to have the `/library` segment in the path, hoping that Docker won't change the setup of the registry.

Reply:
I mean the canonical location is `golang` but the fully qualified location is `docker.io/library/golang:latest`